src/Views/User/User.js

```diff
@@ -92,7 +92,7 @@ const User = ({ accessToken }) => {
             limit={onPageLimit}
           />
         )}
-        {bookList}
+        {bookList.length > 0 ? bookList : "Loading..."}
```

Comment:
what if returned bookList has 0 elements? UI would show "Loading", but everything would be loaded. Am I right? What do you think?